src/librbd/Journal.cc

```diff
@@ -471,6 +624,31 @@ int Journal<I>::get_tag_owner(IoCtx& io_ctx, std::string& image_id,
 }
 
 template <typename I>
+void Journal<I>::is_tag_owner(I *image_ctx, ContextWQ *work_queue,
```

Comment:
Did you still run into an issue reusing journal::OpenRequest/journal::CloseRequest?